test/monitoring/specs/statuspage_group.json

```diff
@@ -11,7 +11,8 @@
     "--email",
     "setup",
     "--silent",
+    "--name",
     "--group",
-    "\"foo\""
+    "foo"
```

Comment:
Specifying a group, only, ends up with an empty `--name` argument, which should probably be omitted